### DISCUSSION TITLE
GPIO 3.5 " installation guide

### DISCUSSION THREAD

**Discussion by dbucha** [UnchartedBull/OctoDash#1428] [General]
Hello, I tried to instal, Octodash with your guide but my LCD screen always shows just white screen, I followed this instructions to get it working:

After updating OctoPrint to the latest version and then rebooting, let's now set up the display. Please follow the following steps:

$ sudo apt-get update

$ sudo apt-get install -f

$ git clone https://github.com/waveshare/LCD-show.git

$ chmod -R 755 LCD-show

$ cd LCD-show/

$ chmod +x LCD35-show

(NOTE: the waveshare drivers are display-specific. Please consult the manufacturer of your display for the video drivers and the install steps)

$ sudo ./LCD35-show

This last step will reboot your system and you will need to SSH back in.

(NOTE: Use sudo ./LCD35-show if you want the orientation of the screen to have the USB ports on the right and the power plug on the bottom, or use sudo ./LCD35-show 180 if you want the orientation of the screen to have the USB ports on the left and the power plug on the top)

Once SSH'd back in, continue with the following steps...

$ sudo nano /boot/config.txt

There should be an existing hdmi_cvt line near the bottom of config.txt. Change the line to:

hdmi_cvt 800 533 60 6 0 0 0

Now reboot... $ sudo reboot now ...and SSH back in.

$ sudo apt-get install libgtk-3-0 xserver-xorg xinit x11-xserver-utils

Now reboot... ...and SSH back in.

$ sudo apt-get install git build-essential xorg-dev xutils-dev x11proto-dri2-dev

Now reboot... ...and SSH back in.

(NOTE: many of these reboots probably aren't necessary, but do them anyway, just to be safe)

$ sudo apt-get install libltdl-dev libtool automake libdrm-dev

Now reboot... ...and SSH back in.

$ git clone https://github.com/ssvb/xf86-video-fbturbo.git

$ cd xf86-video-fbturbo

$ autoreconf -vi

$ ./configure --prefix=/usr

$ make

(NOTE: at this point I got several warnings about "const", and "expected char * but type is const char *". I ignored the warnings and continued on)

$ sudo make install

$ sudo cp xorg.conf /etc/X11/xorg.conf

Now reboot... ...and SSH back in.

I took this guide from the old Octoscreen.
https://github.com/Z-Bolt/OctoScreen/wiki/Installing-OctoScreen-with-a-3.5%22-480x320-TFT-screen

Best regards,

**Comment by UnchartedBull** [comment]
Does your screen always show a white screen only or is there something else visible? If it's just white it's probably hooked up wrong.

**Reply by dbucha** [reply]
It´s always white screen, I have to follow the guide from Octoscreen to get it working.

**Comment by jonathanalevi** [comment]
You're a LEGEND! This got it working for me :)

**Comment by marc-gist** [comment]
Did this, but touch doesn't work :(
